gemfiles/common.rb

```diff
@@ -2,6 +2,3 @@
 source "https://rubygems.org"
 
 gemspec path: Bundler.root.to_s.sub('/gemfiles', '')
```

Comment:
Just change this to `gemspec path: '..'` and you'll get rid of your local path problems in the lockfiles.

Reply:
🎉 thx!